test/danetest.c

```diff
@@ -287,25 +263,22 @@ static int tlsa_import_rr(SSL *ssl, const char *rrdata)
 
     for (f = tlsa_fields; f->var; ++f) {
         if ((len = f->parser(cp += len, f->var)) <= 0) {
-            fprintf(stderr, "%s: warning: bad TLSA %s field in: %s\n",
-                    progname, f->name, rrdata);
+            TEST_info("bad TLSA %s field in: %s", f->name, rrdata);
```

Comment:
Why is this `TEST_info` and not `TEST_error`?  You did change warnings into `TEST_error` up at new lines 162+...